src/Cake.Svn.Tests/Unit/Info/SvnInfoResultTests.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using Cake.Svn.Info;
+using Xunit;
+
+namespace Cake.Svn.Tests.Unit.Info
+{
+    public sealed class SvnInfoResultTests
+    {
+        public sealed class TheConstructor
+        {
+            private SvnInfoResult svnInfoResult = null;
+
+            private Guid guid = Guid.NewGuid();
+            private Uri repositoryRoot = new Uri("https://svn.example.com");
+            private string lastChangedAuthor = "administrator";
+            private long revision = 1234;
+            private Uri uri = new Uri("https://svn.example.com");
+            private string path = @"src/folder/file.cs";
+            private string fullPath = @"https://svn.example.com/src/folder/file.cs";
+            private SvnKind nodeKind = SvnKind.Unknown;
```

Comment:
Having all these private variables is IMHO not a good test design which can lead to dependencies between tests, especially since they are note readonly. Better make a private method which return a `SvnInfoResult`